packages/compartment-mapper/src/bundle-lite.js

```diff
@@ -425,13 +425,7 @@ export const makeFunctorFromMap = async (
     } else {
       module.indexedImports = Object.fromEntries(
         Object.entries(module.resolvedImports).map(([importSpecifier, key]) => {
-          // UNTIL https://github.com/endojs/endo/issues/1514
-          // Prefer: key = aliases.get(key) ?? key;
-          const alias = aliases.get(key);
-          if (alias != null) {
-            key = alias;
-          }
-          const module = modulesByKey[key];
+          const module = modulesByKey[aliases.get(key) ?? key];
           if (module === undefined) {
             throw new Error(
               `Unable to locate module for key ${q(key)} import specifier ${q(
```

Comment:
Not quite equivalent because the before code might use the value of `alias` in the error message and the after code will only ever use `key`. Aside from syntax, which is the better behavior?

Reply:
Neither. Posting an update that varies the error depending on whether an alias was followed to provide a better debugging experience.